media/Config/configValidator.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// validator for input_path and output_path, this validator checks only for empty or not
+const pathValidator = function(tool) {
+  for (let j = 0; j < tool.options.length; j++) {
+    if (tool.options[j].optionName === 'input_path' && tool.options[j].optionValue.trim() === '') {
+      sendMessage('alert', `If you want to use ${tool.type}, then input_path is required`);
```

Comment:
(optional) maybe `input_path is required to use ${tool.type}`

Reply:
Q) will `sendMessage` show popup message through WebView?

Reply:
yes, senndMessage invoke Balloon.error in Utils/Balloon.ts